### ISSUE TITLE
Main panel blank in MacOS High Sierra 10.13.6

### ISSUE THREAD

**Issue by michael-wenyon** [corelan/EncFSGui#8]
### [solved? I had two Menuapps/EncFSGui instances running? see third comment below]

I can use EncFSGui v1.0.139, but the main panel always appears blank (empty, white), even when I use the 'Open Existing' button to add an existing EncFS volume. I can still use EncFSGui because, although the main panel list does appear empty, the new volume is listed in the Menubar app, where the 'Volumes>Mount' and 'Volumes>Unmount' commands both work and are sufficient for me to mount/unmount volumes.

To remove volumes, I have to delete them all by deleting the 'encfsgui Preferences' file in ~/Library/Preferences/, so I would definitely like to be able to use the main window. The syntax of the preferences file does not look too difficult, so perhaps I could delete individual volumes just by deleting their code inside the preferences file.

*MacOS 10.13.6, OSXFuse 3.8.2, encfs 1.9.5 (via homebrew), EncFSGui 1.0.139, xcode-select version 2349, gcc version: Apple LLVM version 9.1.0 (clang-902.0.39.1)
I appear to have the encfs binary installed at the path specified in the Settings panel.*

**Comment by michael-wenyon** [comment]
![encfsgui_empty](https://user-images.githubusercontent.com/5072302/47090245-8647aa80-d1f0-11e8-9312-c9d6ad76ffd1.png)
Screen shot of 'empty' EncFSGui panel. The buttons in the window that do work are: Create, Open Existing, Force Unmount all, Settings and Quit. Others are greyed.

**Comment by michael-wenyon** [comment]
The buttons in the window that *do* work are: Create, Settings and Quit.
I confirmed that I do have the encfs binary installed at the path specified in the Settings panel.

**Comment by michael-wenyon** [comment]
### Problem solved?
I just noticed (after logging out and logging in again) I had **two** icons for EncfsGui in my Menu Bar. I 'quit' the one furthest to the right, then the 'Show EncfsGui' command in the remaining Menubar menu properly revealed the main panel including the saved Volumes settings listed in blue. The panel buttons all seemed to work as expected and I could, for example, mount the volume using the 'Mount & expose' button.

The next time I logged out/in, only one Menubar item appeared for EncfsGui (the one on the left end of the icons) and it continued to reveal the main panel with the list filled and buttons working; the 'fix' also survived a restart of my Mac.

Happy to be able to use this, as a former Boxcryptor customer needing encfs access across platforms. I feel good about using this as a 'wrapper' to encfs, because I can update encfs independently and revert to command-line use of encfs if EncFSGui stops working one day. Indeed, I would suggest anyone trying to use EncFSGui should first install and get comfortable with command-line use of encfs installed via homebrew, it helps to understand the concepts and gives you a fallback technique.

**Comment by michael-wenyon** [comment]
### Postscript: What happened?

Installing on my 2nd Mac, I may have discovered how this may have happened. On first run, EncFSGui is rather quiet: you only know it is there by the menubar icon and as an inexperienced user I did not notice this. Thinking it had crashed or not opened at all, I double-clicked the icon in the Applications folder a second time--possibly starting a second instance of the program? Then, recent MacOS always tries to re-open apps/files you had open when you last logged out, so perhaps it simply reopened EncFSGui two times again, every time I logged in. That would explain why quitting one of the menu icons simply 'fixed' this problem once and for all, because by then I knew enough about EncFSGui not to expect to see the main panel until selecting 'Show EncFSGui' in the menubar menu (and not to open the program twice).

Anyway, I did not do any double double-clicking on Mac 2 and the initial installation worked fine straight away, with the proviso that had I not known this program I might have missed the subtle appearance of the menubar icon on first run.

**Comment by corelanc0d3r** [comment]
wow - interesting experience indeed.  I don't know why the panel would be blank if you launch it twice... but at least I'm happy that you discovered what the issue is (and how to fix it when it happens)

thanks for testing (and sorry for the trouble you wen through)
I know the app is not perfect, and unfortunately I lack the time to make improvements myself

thanks again !

**Comment by michael-wenyon** [comment]
Thank you very much for your work on this, which I am using, not just testing! I will post this as a separate issue concerning simultaneous instances being possible in case any developer can prevent this behavior in future.

**Comment by michael-wenyon** [comment]
Just to add, I got the same symptoms (blank volumes list and EncFSGui hiding itself twice when 'Hide EncFSGui' chosen in menubar) on a new installation on a new Mac. Could it be a problem that only shows on first run of a new installation?

I remember something similar a year or two ago, when I first tried EncFSGui. At the time, I gave up and went back to command-line EncFS use because of this.
